From b9c0bd2193703749901eaebde9e1ac7e31f0cd6f Mon Sep 17 00:00:00 2001
From: rphii
Date: Sat, 15 Mar 2025 13:53:35 +0000
Subject: [PATCH 1/1] vcs_tag

---
 Makefile.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index f6f57740..73ea347e 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -748,7 +748,7 @@ $(eval $(call rules-meson,dxvk,32,CROSS))
 $(eval $(call rules-meson,dxvk,64,CROSS))
 
 $(OBJ)/.dxvk-post-source:
-	sed -re 's#@VCS_TAG@#$(shell git -C $(SRCDIR)/dxvk describe --always --abbrev=15 --dirty=0)#' \
+	sed -re 's#@VCS_TAG@#9999#' \
 	    $(SRCDIR)/dxvk/version.h.in > $(DXVK_SRC)/version.h.in
 	touch $@
 
@@ -857,9 +857,9 @@ $(eval $(call rules-meson,vkd3d-proton,32,CROSS))
 $(eval $(call rules-meson,vkd3d-proton,64,CROSS))
 
 $(OBJ)/.vkd3d-proton-post-source:
-	sed -re 's#@VCS_TAG@#$(shell git -C $(SRCDIR)/vkd3d-proton describe --always --exclude=* --abbrev=15 --dirty=0)#' \
+	sed -re 's#@VCS_TAG@#9999#' \
 	    $(SRCDIR)/vkd3d-proton/vkd3d_build.h.in > $(VKD3D_PROTON_SRC)/vkd3d_build.h.in
-	sed -re 's#@VCS_TAG@#$(shell git -C $(SRCDIR)/vkd3d-proton describe --always --tags --dirty=+)#' \
+	sed -re 's#@VCS_TAG@#9999#' \
 	    $(SRCDIR)/vkd3d-proton/vkd3d_version.h.in > $(VKD3D_PROTON_SRC)/vkd3d_version.h.in
 	touch $@
 
-- 
2.48.1

